critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -23,8 +23,10 @@
     </div>
     <div class="form-group">
       <small class="text-muted pull-left" style="font-style:italic;">
-        {{ _('You can use <a href="%(url)s" target="_blank">Markdown</a> syntax to apply custom formatting.',
-             url='http://daringfireball.net/projects/markdown/syntax') }}
+          {{ _('Make sure to keep the <a href="%(style_guide_url)s" target="_blank">style guide</a> in mind when writing.
```

Comment:
I think we can write the statement such that it doesn't seem as a strict compulsion/constraint. Something like:
"Read the [style guide](https://wiki.musicbrainz.org/How_to_write_Good_Critiquebrainz_Reviews) for writing good CritiqueBrainz reviews!"

Reply:
Amended commit.